src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rental = 40;
```

Comment:
Consider using a constant with a descriptive name instead of a literal for the daily rental rate to adhere to the checklist item 2.